tools/wptrunner/wptrunner/testloader.py

```diff
@@ -504,7 +504,7 @@ def iter_tests(self):
             manifest_items = self.chunker(manifest_items)
 
         for test_type, test_path, tests in manifest_items:
-            manifest_file = manifests_by_url_base[iter(tests).next().url_base]
+            manifest_file = manifests_by_url_base["/"]
```

Comment:
This is obviously wrong. However, given `item.url_base` was previously just the constant `"/"`, this isn't a regression. I presume @jgraham will say I need to fix this code though. )

Reply:
You need to fix this code. Or change the iterator to produce something different like `(manifest, source_file, tests)`.

Reply:
So, we only get this right when loading the JSON, not when actually creating the manifest.